ext/fetch/lib.rs

```diff
@@ -514,8 +546,10 @@ pub async fn op_fetch_send(
       // TODO(mmastrac): it would be a lot easier if we just passed a v8::Global through here instead
       let mut err_ref: &dyn std::error::Error = err.as_ref();
       while let Some(err) = std::error::Error::source(err_ref) {
-        if let Some(err) = err.downcast_ref::<reqwest::Error>() {
-          if err.is_body() {
+        // TODO(sean)
+        if let Some(err) = err.downcast_ref::<hyper::Error>() {
+          if true {
```

Comment:
Should we address this before landing?

Reply:
Ah, yea. So, question. Is this looking for a specific source type? The `is_body()` check is gone in hyper 1.0. But if the source is always something from Deno, we can check for that? Or perhaps just any source from a hyper error is what you really want?

Reply:
No, I don't think we were looking for specific source type. The difference seems to be that "non-body" errors were causing `TypeError` to be returned, while other "body" errors where causing a specific error to be returned. I think we might be able to remove this branch altogether

Reply:
Looks like removing this block causes `fetchRequestBodyErrorCatchable` to fail.

Reply:
Yeah, after deeper look it appears we need to handle this more gracefully. Is there any other condition(s) we can check that would emulate what `Error.is_body()` did in the past?

Reply:
It looks like this is meant to capture the error coming from `ResourceToBodyAdapter`. If so, I can adjust that to wrap the user's error in a specific type, and then look for it when down casting.

Reply:
Ah that makes sense, thanks!